collections.md

```diff
@@ -165,5 +144,30 @@ Collection::times(7, function ($number) {
 // Output: [01-04-2022, 02-04-2022, ..., 07-04-2022]
 ```
 
-Tip given by [@Teacoders](https://twitter.com/Teacoders/status/1509447909602906116)
+### Paginate from array method
+
+This method enables you to do all things that the official paginate do but with Arrays.
 
+```php
+use Illuminate\Pagination\LengthAwarePaginator;
+```
+
+```php
+public function paginate($perPage, $total = null, $page = null, $pageName = 'page')
+    {
+        $page = $page ?: LengthAwarePaginator::resolveCurrentPage($pageName);
+
+        return new LengthAwarePaginator(
+            $this->forPage($page, $perPage),
+            $total ?: $this->count(),
+            $perPage,
+            $page,
+            [
+                'path' => LengthAwarePaginator::resolveCurrentPath(),
+                'pageName' => $pageName,
+            ]
+        );
+    }
+```
+
+Tip given by [@Teacoders](https://twitter.com/Teacoders/status/1509447909602906116)
```

Comment:
This tip is currently incomplete and does not contain enough information to inform someone how to paginate an array. I've seen this exact code elsewhere and it contains a more complete example of the class implementation.  In its current form it cannot be used to directly paginate an array.  Additionally, there probably aren't a bunch of people using custom Collections, so the macro method is probably more helpful. In either case, it would have been appropriate to link your source.

Your tip also pushes the previous tips attribution line out of place.

```suggestion
Tip given by [@Teacoders](https://twitter.com/Teacoders/status/1509447909602906116)

### Paginated Collections

Add this to you AppServiceProvider.php files boot() method, or similar provider, and you can then paginate any collection or array with ease. 

```php
        use Illuminate\Pagination\LengthAwarePaginator;

        /**
         * Paginate a standard Laravel Collection.
         *
         * @param  int  $perPage
         * @param  int  $total
         * @param  int  $page
         * @param  string  $pageName
         * @return LengthAwarePaginator
         * @source https://gist.github.com/simonhamp/549e8821946e2c40a617c85d2cf5af5e
         */
        Collection::macro('paginate', function ($perPage, $total = null, $page = null, $pageName = 'page') {
            $page = $page ?: LengthAwarePaginator::resolveCurrentPage($pageName);

            /** @var Collection $this */
            return new LengthAwarePaginator(
                $this->forPage($page, $perPage),
                $total ?: $this->count(),
                $perPage,
                $page,
                [
                    'path' => LengthAwarePaginator::resolveCurrentPath(),
                    'pageName' => $pageName,
                ]
            );
        });
```

Example Usage: 
`collect([ ... ])->paginate( 20 );`

```